vacations/__init__.py

```diff
@@ -85,7 +86,7 @@ def get_province_codes(self):
 
     def get_vacation(self, date_str=None, province=None):
         if province is None:
-            if self.shtime.config.get('location', None).get('country') not in ['DE', 'de']:
+            if self.shtime.config.get('location', None).get('country') not in ['DE', 'de', 'Germany']:
```

Comment:
would it make sense to check for "germany" and "Germany" here? I am not 100% sure anymore where the value comes from. is it resolved by lat and lon? or custom set? or can it be both?

Reply:
ah i see its from location.yaml and "Germany" is listed as valid one.. in any case i think "germany" makes also sense here

Reply:
country is converted to lowercases now, so it is not case-sensitive